devdox/app/services/git_tokens_service.py

```
@@ -130,122 +128,73 @@ def mask_token(token: str) -> str:
+            crypto_store: Annotated[FernetEncryptionHelper, Depends(get_encryption_helper)],
+		    git_manager: Annotated[RepoFetcher, Depends()]
     ) -> "PostGitLabelService":
-        return cls(user_store, crypto_store)
+        return cls(
+            user_store=user_store,
+            label_store=label_store,
+            crypto_store=crypto_store,
+            git_manager=git_manager
+        )
 
     async def add_git_token(self, user_claims:UserClaims, json_payload: GitLabelBase):
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)